extensions/wikia/Email/Controller/WatchedPageController.class.php

```diff
@@ -25,7 +25,9 @@ public function getSubject() {
 
 	// TODO setup error handling here if minorEdit not passed in
 	public function initEmail() {
-		$this->title = $this->request->getVal( 'title' );
+		//$this->title = $this->request->getVal( 'title' );
```

Comment:
Thanks, yeah, I submitted this in a bit of a hurry last night. I'll clean up the weirdness.